src/components/tx/CreateTx/create.js

```diff
@@ -75,23 +75,32 @@ const CreateTx = connect(
     const fiatRate = state.wallet.settings.get('localeRate');
     const fiatCurrency = state.wallet.settings.get('localeCurrency');
 
-    const value = (selector(state, 'value')) ? selector(state, 'value') : 0;
-
     const useLedger = account.get('hardware', false);
     const ledgerConnected = state.ledger.get('connected');
 
     const gasLimit = selector(state, 'gas') ? new BigNumber(selector(state, 'gas')) : new BigNumber(DefaultGas);
+
+    let value = (selector(state, 'value')) ? selector(state, 'value') : 0;
+    let to;
+    // Transaction is passed in if this is a repeat transaction
+    if (ownProps.transaction) {
+      value = new Wei(ownProps.transaction.get('value')).getEther();
+      to = ownProps.toAccount.get('id');
```

Comment:
There is a problem here. `ownProps.toAccount` can be `null`. This is the case when `to` is not your account, but external one. As a solution I can suggest pass pure account id (address) as a string to CreateTx component